Games/types/Mafia/roles/cards/KillVisitorsWhileDead.js

```diff
@@ -0,0 +1,52 @@
+const Card = require("../../Card");
+const { PRIORITY_KILL_VISITORS_ENQUEUE_DEAD, PRIORITY_KILL_VISITORS } = require("../../const/Priority");
+
+module.exports = class KillVisitorsWhileDead extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.actions = [
+            {
+                priority: PRIORITY_KILL_VISITORS_ENQUEUE_DEAD,

```

Comment:
You can now just add the "EnqueueVisitors" card, refer to babushka/ LM. It will enqueue visitors regardless of dead or alive

If there are performance issues after deployment, we can address adding "whileDead" to actions